Dockerfile

```diff
@@ -1,12 +1,11 @@
-# common base image for development and production
-FROM node:10.18.0-alpine AS base
+FROM node:12-alpine
 RUN apk add --no-cache bash
 WORKDIR /app
 
 COPY package.json yarn.lock ./
 RUN yarn install --frozen-lockfile && yarn cache clean
 
-COPY . .
+COPY ./__tests__ ./sql ./src ./perfTest ./tsconfig.json ./
```

Comment:
Does this do the right thing? Docker `COPY` notes:

> If <src> is a directory, the entire contents of the directory are copied, including filesystem metadata.
> Note: The directory itself is not copied, just its contents.

but also:

> If multiple <src> resources are specified, either directly or due to the use of a wildcard, then <dest> must be a directory, and it must end with a slash /.

https://docs.docker.com/engine/reference/builder/#copy

Initially I thought this would be equivalent to:

```
cp ./__tests__/* ./sql/* ./src/* ./perfTest/* ./tsconfig.json DESTINATION
```

i.e. everything gets smooshed into one folder? But does this only happen when there's exactly one `src` argument (which is a directory), and if there's multiple src arguments then the behaviour differs? (This seems highly confusing!)

Reply:
Hmm, good catch, I missed that somehow. FWIW I think just using `COPY . .` is better, it saves having to duplicate the paths, and given this is an image for local dev only I don't see it being an issue. But I'll separate out into separate `COPY` statements